docs/core/docker/intro-net-docker.md

```diff
@@ -1,191 +1,66 @@
-
-* [Run the Microsoft SQL Server for Linux 2017 container image with Docker Quickstart](https://docs.microsoft.com/sql/linux/quickstart-install-connect-docker)
-
-* [Microsoft SQL Server for Linux images on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-linux/)
-
-* [Microsoft SQL Server Express Edition images for Windows Containers on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-windows-express/)
-
-* [Microsoft SQL Server Developer Edition images for Windows Containers on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-windows-developer/)
-
-#### Azure DevOps Services agent
-
-* [Azure DevOps Services agent images on DockerHub](https://hub.docker.com/r/microsoft/vsts-agent/)
-
-* [Azure DevOps Services agent images on GitHub](https://github.com/Microsoft/vsts-agent-docker)
-
-#### Operations Management Suite (OMS) Linux agent
-
-* [Operations Management Suite (OMS) Linux agent overview](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/Docker-Instructions.md)
+The [relationship between Docker containers, images, and registries](../../standard/microservices-architecture/container-docker-introduction/docker-containers-images-registries.md) is an important concept when [architecting and building containerized applications or microservices](../../standard/microservices-architecture/architect-microservice-container-applications/index.md). This approach greatly shortens the time between development and deployment.
 
-* [Operations Management Suite (OMS) images on DockerHub](https://hub.docker.com/r/microsoft/oms/)
+### Dockerfile
 
-* [Operations Management Suite (OMS) images on GitHub](https://github.com/Microsoft/OMS-docker)
+A **Dockerfile** is a file that defines a set of instructions that creates an image. Each instruction in the **Dockerfile** creates a layer in the image. When you rebuild the image, only the layers that have changed are rebuilt. The **Dockerfile** can be distributed to others and allows them to recreate the image on their own system.
 
-#### Microsoft Azure Command Line Interface (CLI)
+## .NET Core images
 
-* [Microsoft Azure Command Line Interface (CLI) images on DockerHub](https://hub.docker.com/r/microsoft/azure-cli/) 
+Official .NET Core Docker images are available at the [Microsoft .NET Core Docker Hub repository](https://hub.docker.com/_/microsoft-dotnet-core/). Each repository contains multiple images for different combinations of the .NET (SDK or Runtime) and OS that you can use. 
 
-* [Microsoft Azure Command-Line Interface (CLI) images on GitHub](https://github.com/Azure/azure-cli#Docker)
+Microsoft provides images that are tailored for specific scenarios. For example, the [ASP.NET Core repository](https://hub.docker.com/_/microsoft-dotnet-core-aspnet/) provides images that are built for running ASP.NET Core apps in production. You can use images that include only the Runtime omitting the SDK so that you can mirror a production environment.
```

Comment:
> You can use images that include only the Runtime omitting the SDK so that you can mirror a production environment.

What is the core point trying to be conveyed here?  I find "omitting the SDK" and "mirror a production environment" to be strange.  A runtime image can be a production environment.